admin/acceptance.py

```diff
@@ -305,6 +343,8 @@ def main(args, base_path, top_level):
 
     runner = options.runner
 
+    signal.signal(signal.SIGTERM, signal_handler)
```

Comment:
It might be nice to walk a future maintainer through the logic here a little bit.  Signals can be subtle (and I'll admit that I'm not 100% sure I understand what's going to happen when a `SIGTERM` is delivered to this process).